next/components/forms/widget-components/Modals/Modal.tsx

```diff
@@ -104,6 +104,10 @@ const ModalFooter = ({
       >
         <span aria-hidden="true">{cancelLabel}</span>
       </button>
+      MODALITO??
+      <Button variant="black-solid" size="small" onPress={onClose}>
+        <span aria-hidden="true">{cancelLabel}</span>
+      </Button>
```

Comment:
Why there is span inside the Button please? It should not be needed.

Reply:
I must have left it there when using new button. Will delete. But, I've noticed that this modal is not used. It's imported to AccountModal but this modal isn't imported anywhere. Only modal made from RAC components is used  Should we keep it?

Reply:
You can remove unused file.